src/plugins/data-picture/data-picture-en.hbs

```diff
@@ -27,16 +27,16 @@
 				<!-- Default image: <div [data-src]> with no data-media attribute is displayed when:
 					1. None of the other <div [data-src]> media queries match.
 					2. The browser doesn't support media queries -->
-				<span data-src="img/parliament-hill-large.jpg"></span>
+				<span data-src="demo\img\parliament-hill-large.jpg"></span>
```

Comment:
Should these be forward slashes (``/``) instead of backslashes (``\``)? Looks like most of this PR's changes introduce backslashes.

Backslashes aren't a common convention in URLs. AFAIK they're usually only used in Windows file paths.